test/TensorFlow/integration.swift

```diff
@@ -239,15 +230,14 @@ public func test_multiple_ifs(status: Bool) {
   _hostOp(a)
 }
 
+// The results are different here because compiler optimizes well
```

Comment:
@mhong please make sure this is fine.

Reply:
hmm the two if's got merged, probably by the performance pipeline -- if so we'll (temporarily) lose such optimization once we move up the partition pass, and cause another test diff.

So it might be better to rewrite this test with 2 different cond values, like `public func test_multiple_ifs(status1: Bool, status2: Bool)`

(It could be useful to have separate tests on what the optimizer can do, but this "integration" test suite is not the best place for that.)

Reply:
Done. I just changed the second condition instead of adding a new variable.